core/block_test.go

```diff
@@ -204,12 +126,12 @@ func TestBlockHash(t *testing.T) {
 	})
 
 	t.Run("no error if block is unverifiable", func(t *testing.T) {
```

Comment:
new version of this test doesn't test what it was intended to. Here is metadata of goerli:
```
UnverifiableRange:        []uint64{119802, 148428},
```
119801 is beginning of unverifiable range

Reply:
If we don't need this logic anymore it's even better, but I'm not sure. cc @IronGauntlets